web/templatetags/markdown_filters.py

```
@@ -1,11 +1,54 @@
+    # Convert markdown to HTML
+    html = markdownify(content)
+
+    # Sanitize HTML
+    clean_html = bleach.clean(
+        html,
+        tags=SAFE_TAGS,
+        attributes=SAFE_ATTRIBUTES,
+        strip=True
+    )
+
+    return clean_html
```

Ruff (0.15.6)

[warning] Unnecessary assignment to `clean_html` before `return` statement

Remove unnecessary assignment

(RET504)